### ISSUE TITLE
Sanitize names

### ISSUE THREAD

**Issue by jsejcksn** [WebReflection/emoji-essential#1]
[The names of recently-added emoji include indicator symbols: `⊛`](https://unicode.org/emoji/charts/full-emoji-list.html#:~:text=Recently-added%20emoji%20are%20marked%20by%20a%20%E2%8A%9B%20in%20the%20name)

Example: https://github.com/WebReflection/emoji-essential/blob/6402a83ce9a9c071dfc93ce1ca2385eeb27f9052/index.js#L9

It looks like you're scraping these. Unicode offers [a "machine-readable" text format](https://unicode.org/Public/emoji/latest/emoji-test.txt) if you're interested. You don't seem to be scraping keywords or other data only available in the html pages, so it might be a better fit for this project.

If you _are_ interested in adding keywords to the dictionary, [this page](https://unicode.org/emoji/charts/emoji-list.html) might interest you.

---

Off-topic, but mildly-related: If you prefer `electron` scraping to `puppeteer`, care to share why? I've never seen someone use `electron` for that purpose, and I'm interested.

**Comment by rdela** [comment]
Yes as @jsejcksn wrote the ⊛ character is not part of the name: “[Recently-added emoji](https://www.unicode.org/emoji/charts/emoji-released.html) are marked by a ⊛ in the name” - [Full Emoji List, v15.1](https://www.unicode.org/emoji/charts/full-emoji-list.html)

@WebReflection are you interested in implementing this change? or open to a PR? I am using https://github.com/WebReflection/emoji-short-name via the [`accessible-emoji` component](https://github.com/11ty/tugboat/blob/main/_components/accessible-emoji.webc) in [Tugboat](https://tugboat.11ty.dev/) and the ⊛ in the names is a big drag here and defeats the whole purpose of trying to increase access, Cc @zachleat

UPDATE: I added a workaround in [webcbed #5](https://github.com/rdela/webcbed/pull/5/files). (EDIT: and [webcbed #6](https://github.com/rdela/webcbed/pull/6/files))
[Current code](https://github.com/rdela/webcbed/blob/trunk/_components/accessible-emoji.webc)

UPDATE 2: Seems like Mac speech ignores ⊛ but I am sure some devices read “CIRCLED ASTERISK OPERATOR” out loud.
